tests/sqlparser_clickhouse.rs

```diff
@@ -220,6 +220,40 @@ fn parse_create_table() {
     );
 }
 
+#[test]
+fn parse_create_view_with_fields_data_types() {
+    match clickhouse().one_statement_parses_to(
```

Comment:
Can we add a test case that clickhouse fails parsing if the datatype is missing?

Reply:
Makes sense.
Added a test with invalid query `CREATE VIEW` without data types.